AsyncDisplayKit/Details/ASDataController.h

```diff
@@ -116,6 +117,16 @@ extern NSString * const ASCollectionInvalidUpdateException;
 - (instancetype)initWithDataSource:(id<ASDataControllerSource>)dataSource eventLog:(nullable ASEventLog *)eventLog NS_DESIGNATED_INITIALIZER;
 
 /**
+ * The map that is currently displayed. The "UIKit index space."
+ */
+@property (nonatomic, strong, readonly) ASElementMap *visibleMap;
```

Comment:
I wonder if there's a better name for this… I feel like completeMap is wrong but possibly is less confusing (maybe it's just confusing in a different way). Possibly uiSpaceMap (looks terrible)?